parser-typechecker/src/Unison/FileParsers.hs

```diff
@@ -176,9 +176,8 @@ synthesizeFile ambient tl fqnsByShortName uf term = do
         -- use tlcsFromTypechecker to inform annotation-stripping decisions
         traverse (traverse strippedTopLevelBinding) tlcsFromTypechecker
     let doTdnr = applyTdnrDecisions infos
-        doTdnrInComponent (v, t, tp) = (\t -> (v, t, tp)) <$> doTdnr t
-    _          <- doTdnr tdnrTerm
```

Comment:
`doTdnr` is pure, so this seems to have been doing nothing?

Reply:
I'm not too familiar with this code, but you could check with @runarorama or try to dig it out of git history.